rec_to_nwb/processing/builder/originators/position_originator.py

```diff
@@ -53,15 +53,23 @@ def make(self, nwb_content: NWBFile):
                 position_df = self.get_position_with_corrected_timestamps(
                     position_tracking_path[0]
                 )
-                position.create_spatial_series(
-                    name=f"series_{dataset_ind}",
-                    description=", ".join(position_df.columns.tolist()),
-                    data=np.asarray(position_df),
-                    conversion=conversion,
-                    reference_frame="Upper left corner of video frame",
-                    timestamps=np.asarray(position_df.index),
-                )
-                first_timestamps.append(position_df.index[0])
+                #Multi-position split. 
+                #TODO: generalize key names?
+                key_lists = [['xloc','yloc',],  #led 0 
+                             ['xloc2','yloc2',],] #led 1
+                led_number = 0
+                for led_number, valid_keys in enumerate(key_lists):
+                    key_set = [key for key in position_df.columns.tolist() if key in valid_keys]
```

Comment:
I seem to remember there also being a video frame index in this dataframe. Can you double check that? We might possibly want to store that in a different nwb object.

Reply:
Yes, that information is in the dataframe when it arrives here, but I think nwb format doesn't want that stored in the SpatialSeries object. Is there a logical place for you or spyglass where that should be stored at?

Reply:
@rly do you have thoughts on where best to store these? I apologize because I think I asked you this before and forgot.

Reply:
The short answer is that we could extend the `SpatialSeries` NWB data type to store the video frame indices. Since we already have an extension, it would not be that big of a change. A hacky but functional alternative is to create a `TimeSeries` for each `SpatialSeries` that shares the same timestamps as the `SpatialSeries` and stores the video frame indices in its `data` field. Both options would be most useful to users if the video file is linked to (e.g., as an `ImageSeries`) from these objects, but I think currently, the video file is either not included in the NWB file or removed before upload to DANDI.